src/fsharp/fsc.fs

```diff
@@ -699,17 +718,16 @@ module XmlDocWriter =
 // cmd line - option state
 //----------------------------------------------------------------------------
 
-let getModuleFileName() = 
-    Path.Combine(System.AppDomain.CurrentDomain.BaseDirectory,
-                           System.AppDomain.CurrentDomain.FriendlyName)  
-
-let defaultFSharpBinariesDir = Filename.directoryName (getModuleFileName())
 
+let defaultFSharpBinariesDir = 
+#if FX_RESHAPED_APPCONTEXT
```

Comment:
Yes BaseDirectory has been moved from AppDomain to AppContext.  I think when I typed reshaped appcontext, I imagined AppContext was a lot bigger than it actually was.  FX_NO_APP_DOMAINS seems a decent way to go with this.